templates/addons/cluster-resource-set/secret.yaml

```diff
@@ -14,6 +14,21 @@ stringData:
       apiToken: ${LINODE_TOKEN}
       region: ${LINODE_REGION}
 ---
+apiVersion: v1
```

Comment:
is there a benefit of using an extra Secret instead of adding a new field in the other Secret's stringData?

Reply:
I was having issues setting the variable with cert content and then using it directly as `\n` were messing up things. So I created another secret with `data` instead of `stringdata` and provided the base64 encoded cert there. Not sure if there is a way I can pass cert to original secret without messing up things with formatting here.

Reply:
Sorry, specifically I meant doing:
```
apiVersion: v1
kind: Secret
type: addons.cluster.x-k8s.io/resource-set
metadata:
  name: linode-${CLUSTER_NAME}-crs-0
stringData:
  linode-token-region.yaml: |-
    kind: Secret
    apiVersion: v1
    metadata:
      name: linode-token-region
      namespace: kube-system
    stringData:
      apiToken: ${LINODE_TOKEN}
      region: ${LINODE_REGION}
  linode-ca.yaml: |-
    kind: Secret
    apiVersion: v1
    metadata:
      name: linode-ca
      namespace: kube-system
    data:
      cacert.pem: ${LINODE_CA_BASE64:=""}
```
So then we don't need a `linode-${CLUSTER_NAME}-crs-1`

Reply:
Good point, I missed it. Updated the PR with the change.